assignments/E0.md

```diff
@@ -28,44 +28,44 @@ We would recommend adding at least an additional 20G of space, for a total of 50
 
 If you run out of disk space in the middle of compilation, you'll almost certainly have to
 delete files to free up enough space to resize your filesystem, which would likely
-mean running `make clean` and losing your progress in the compilation.
+mean running `$ make clean` and losing your progress in the compilation.
```

Comment:
I think it is better to leave off the prefix when the command is inline within a paragraph